src/shape/bounding-sphere.js

```diff
@@ -3,6 +3,7 @@ pc.extend(pc, function () {
     var tmpVecB = new pc.Vec3();
     var tmpVecC = new pc.Vec3();
     var tmpVecD = new pc.Vec3();
+    var diffBetweenPoints = new pc.Vec3();
```

Comment:
Yeah, don't worry.. The functionA was just a made up function name for the example.

Fair enough, changing it now.